src/Classes/ModStore.lua

```diff
@@ -501,8 +501,36 @@ function ModStoreClass:EvalMod(mod, cfg)
 				return
 			end
 		elseif tag.type == "SocketedIn" then
-			if not cfg or tag.slotName ~= cfg.slotName or (tag.keyword and (not cfg or not cfg.skillGem or not calcLib.gemIsType(cfg.skillGem, tag.keyword))) then
+			if not cfg or tag.slotName ~= cfg.slotName or (tag.socketColor ~= nil and (tag.sockets == nil or (tag.sockets ~= nil and type(tag.sockets) ~= "string")) and tag.socketColor ~= cfg.socketColor) or (tag.keyword and (not cfg or not cfg.skillGem or not calcLib.gemIsType(cfg.skillGem, tag.keyword))) then
 				return
+			elseif tag.sockets ~= nil and tag.sockets == "all" and tag.socketColor ~= nil then
+					local targetAtrColor = tag.socketColor == "R" and "strengthGems" or tag.socketColor == "G" and "dexterityGems" or tag.socketColor == "B" and "intelligenceGems"
+					local total = (cfg["intelligenceGems"] or 0) + (cfg["dexterityGems"] or 0) + (cfg["strengthGems"] or 0)
```

Comment:
```suggestion
					local total = cfg.intelligenceGems + cfg.dexterityGems + cfg.strengthGems
```
This doesn't need to be a key it can just be cfg.intelligenceGems, etc
You also ensured that these are non nil when you generated them so or 0 is unneccesary.